forms/serializers/form.py

```diff
@@ -291,12 +291,18 @@ class Meta:
         ]
 
     def get_information_checks(self, obj):
-        entry_sections = obj.entry_sections.filter(identifier="hakijan-tiedot")
+        entry_sections = obj.entry_sections.filter(
+            identifier__contains="hakijan-tiedot"
```

Comment:
Realistically, this is fine as is, but changing this to `identifier__startswith="hakijan-tiedot["` would make this even more bulletproof.